src/Lykke.Job.BlockchainCashinDetector.AzureRepositories/DepositBalanceDetectionsDeduplicationRepository.cs

```diff
@@ -52,7 +60,14 @@ private DepositBalanceDetectionsDeduplicationRepository(INoSQLTableStorage<Depos
                 RowKey = DepositBalanceDetectionsDeduplicationEntity.GetRowKey(depositWalletAddress)
             };
 
-            await _storage.InsertOrReplaceAsync(entity);
+            await _storage.InsertOrModifyAsync
```

Comment:
Update Lykke.AzureStorage up to the v6.0.0 and use ```InsertOrReplaceAsync``` with ```replaceCondition``` instead of this obsolete ```InsertOrModifyAsync```